### DISCUSSION TITLE
JSON file from Illumina Nirvana

### DISCUSSION THREAD

**Discussion by NourMarzouka** [labsquare/CuteVariant-python#388] [Ideas]
I wonder if there is any plan to add an import option for the annotated JSON file from the free open-source Nirvana tool?
It is faster than VEP but the issue is the output format is a JSON file only.

**Comment by dridk** [comment]
could you give me the file specification?

**Reply by dridk** [reply]
this is a good idea. Lets do it

**Comment by NourMarzouka** [comment]
Sure!
Nirvana JSON File Format Documentation
https://illumina.github.io/NirvanaDocumentation/file-formats/nirvana-json-file-format

**Comment by dridk** [comment]
Really interesting format. I will add as issue for the next release .

Dr Sacha Schutz
médecine / génétique moléculaire
Bioinformatique
[Clef plublic GPG](http://keys.gnupg.net/pks/lookup?op=get&search=0x214A030FFE54058C)
dridk.me
[My github](https://github.com/)

-------- Message d'origine --------
